service/src/main/java/bio/terra/workspace/service/workspace/AwsCloudContextService.java

```diff
@@ -86,20 +96,47 @@ public void addCreateCloudContextSteps(
       UUID workspaceUuid,
       SpendProfile spendProfile,
       AuthenticatedUserRequest userRequest) {
+    if (featureService.isFeatureEnabled(
+        FeatureService.AWS_APPLICATIONS_ENABLED,
+        flightBeanBag.getSamService().getSamUser(userRequest).getEmail())) {
```

Comment:
Actually, this will not be a per-user feature flag, as it is only here to gate.  We will only gate by user at the AWS platform level.  I will fix this up.